client/datasync-client/package.json

```diff
@@ -33,24 +33,24 @@
     "branches": 50
   },
   "devDependencies": {
-    "@types/chai": "^4.0.1",
-    "@types/lodash": "^4.14.70",
+    "@types/chai": "^4.0.3",
```

Comment:
It will be nice to get some github dependency checker to make sure we always stay up to date. 

Did you updated this manually?